modules/components/pages/outputs/quest_db.adoc

```diff
@@ -0,0 +1,423 @@
+= questdb
+:type: output
+:status: experimental
```

Comment:
If experimental is now considered beta, we can replace this with :page-beta: true

Reply:
Updated as requested. Will circle back on new connectors to apply this change where necessary.